git/repo/base.py

```diff
@@ -697,6 +697,19 @@ def _get_untracked_files(self, *args, **kwargs):
         finalize_process(proc)
         return untracked_files
 
+    def get_ignored(self, *paths):
```

Comment:
This method should be `ignored`, as there is no `get_` prefix for the other getters either.